SocialFilter/sections/specific.txt

```diff
@@ -72,6 +72,7 @@ ibrala.com##.mpp-container
 ibrala.com##a[target="_blank"][rel="noopener"]:not([href*="ibrala.com"]) > img
 hotline.ua##.matrix-head > div.item > div.m_b-md + div.site-service-tools
 yenicaggazetesi.com.tr##.bottom_social
+udn.com##.shareBar__info--push
```

Comment:
It's ok for mobile version, but what about desktop one?
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/109471478-dce07700-7a81-11eb-94ea-a7dea5993e07.png)

</details><br/>